README.md

```diff
@@ -4,7 +4,7 @@
 [![Chrono GitHub Actions][gh-image]][gh-checks]
 [![Chrono on crates.io][cratesio-image]][cratesio]
 [![Chrono on docs.rs][docsrs-image]][docsrs]
-[![Join the chat at https://gitter.im/chrono-rs/chrono][gitter-image]][gitter]
+[![Chat][gitter-image]][gitter]
```

Comment:
I don't know just about anything about chat platforms since irc... Is gitter in some way related to Discord?

Reply:
Nope, they're unrelated. Gitter was acquired by Matrix at some point. There are ways to bridge Matrix and Discord but the bridges can be pretty flaky.

Reply:
They `gitter` should be replaced with `discord`?

Reply:
Ahh, sorry, yes.